utils/entrypoint.sh

```diff
@@ -4,11 +4,24 @@
 # that doesn't exist in /etc/passwd, but Ansible module utils
 # require a named user. So if we're in OpenShift, we need to make
 # one before Ansible runs.
-if [ `id -u` -ge 500 ] || [ -z "${CURRENT_UID}" ]; then
+if [[ (`id -u` -ge 500 || -z "${CURRENT_UID}") ]]; then
+
+    # Only needed for RHEL 8. Try deleting this conditional (not the code)
+    # sometime in the future. Seems to be fixed on Fedora 32
+    # If we are running in rootless podman, this file cannot be overwritten
+    ROOTLESS_MODE=$(cat /proc/self/uid_map | head -n1 | awk '{ print $2; }')
+    if [[ "$ROOTLESS_MODE" -eq "0" ]]; then
 cat << EOF > /etc/passwd
 root:x:0:0:root:/root:/bin/bash
 runner:x:`id -u`:`id -g`:,,,:/home/runner:/bin/bash
 EOF
```

Comment:
the indentation of this is off.

Reply:
We talked about an alternative option of

```bash
cat 'root:x:0:0:root:/root:/bin/bash' >> /etc/passwd
```

But I didn't understand the `cat` command use here, and there are pitfalls, so we need to test image building in separate branch.